posthog/hogql/database/schema/event_sessions.py

```diff
@@ -0,0 +1,126 @@
+from copy import deepcopy
+from typing import Any, Dict, List
+from posthog.hogql import ast
+from posthog.hogql.context import HogQLContext
+from posthog.hogql.database.models import FieldOrTable, IntegerDatabaseField, StringDatabaseField, VirtualTable
+from posthog.hogql.parser import parse_select
+from posthog.hogql.resolver_utils import get_long_table_name, lookup_field_by_name
+from posthog.hogql.visitor import CloningVisitor, TraversingVisitor
+
+
+class EventsSessionSubTable(VirtualTable):
+    fields: Dict[str, FieldOrTable] = {
+        "$session_id": StringDatabaseField(name="$session_id"),
+        "session_duration": IntegerDatabaseField(name="session_duration"),
```

Comment:
What about calling these just `id` and `duration`? 

```sql
SELECT uuid, event, session.session_duration, session.$session_id FROM events LIMIT 100
```
vs
```sql
SELECT uuid, event, session.duration, session.id FROM events LIMIT 100
```

Reply:
I did try this before, but it didn't work - it couldn't resolve the fields. Think this may be related to the same reason the table is showing up as a reference and not a virtual table above. Will investigate